driver/src/main/java/org/neo4j/driver/v1/Config.java

```diff
@@ -84,9 +87,19 @@ public Logging logging()
      * Max number of connections per URL for this driver.
      * @return the max number of connections
      */
+    @Deprecated
     public int connectionPoolSize()
     {
-        return connectionPoolSize;
+        return maxIdleConnectionPoolSize;
+    }
+
+    /**
+     * Max number of idle connections per URL for this driver.
+     * @return the max number of connections
+     */
+    public int maxIdleConnectionPoolSize()
```

Comment:
Since we're deprecating the other `connectionPoolSize`, and the config option on the builder uses the word `session`, could we stealthily swap to use `session` here as well?